cmd/workspace/apps/apps.go

```diff
@@ -187,7 +187,7 @@ func newGet() *cobra.Command {
 	cmd.Use = "get NAME"
 	cmd.Short = `Get definition for an application.`
 	cmd.Long = `Get definition for an application.
-  
+
```

Comment:
Please remove these -- the file is autogenerated, so the diff will show up again in the next commit.